bundle/manifests/liberty.websphere.ibm.com_webspherelibertyapplications.yaml

```diff
@@ -51,7 +51,7 @@ spec:
     name: v1
     schema:
       openAPIV3Schema:
-        description: Represents the deployment of an Open Liberty application
+        description: Represents the deployment of an WebSphere Liberty application
```

Comment:
Should be "deployment of a WebSphere".
Also, this description come from the comment in API ([here](https://github.com/WASdev/websphere-liberty-operator/blob/b85ca38c2fe8f2910f350182139f239af6d2668c/api/v1/webspherelibertyapplication_types.go#L349)), so should be modified there and then `make generate manifests` should be called.